lib/libtlspool.c

```diff
@@ -37,6 +38,18 @@ static pthread_cond_t updated_poolfd = PTHREAD_COND_INITIALIZER;
 
 static pthread_mutex_t prng_lock = PTHREAD_MUTEX_INITIALIZER;
 
+int tlspool_getpid () {
+	int fd;
+
+	if ((fd = open("/var/run/tlspool.pid", O_RDONLY)) != -1) {
+		char str_pid[256];
+
+		if (read(fd, str_pid, sizeof(str_pid)) != -1) {
```

Comment:
Hoi,

Getest: 7310cea45c173ef9e9803c4a87fa0706e74cfee8 "fix 2"

Hij bouwt en alles start op Linux; alleen breken testsrv en testcli de connectie af met "broken pipe" wat het verwachte gedrag is door de aanpassing van SOCK_SEQPACKET naar SOCK_STREAM

Gek genoeg krijg ik ook nog "broken pipe" op 58efc944a82b632d25f7fd95b5cfa1de441d982a

Zoekzoek... verdraaid, die SEQPACKET is helemaal niet van belang op die plek, dat mag best SOCK_STREAM zijn lijkt me, want dat is het socketpair().  Blijkbaar biedt een Pull Request toch wat weinig context om op te oordelen...!

Dan ligt er dus nog een puzzeltje, wat er veranderd kan zijn aan de code...

-Rick